docs/model-hub/transformers/examples.txt

```diff
@@ -0,0 +1,107 @@
+.. _model-hub-transformers-examples:
+
+##########
+ Examples
+##########
+
+`Transformers examples
+<https://github.com/huggingface/transformers/tree/master/examples>`_ are
+the starting point for many users of the `transformers
+<https://github.com/huggingface/transformers>`_ library. Hence, they are
+the core feature of **model-hub's** support for transformers library. In
+fact, each `transformer example corresponding to a core transformer task
+<https://huggingface.co/transformers/examples.html#the-big-table-of-tasks>`_
+has an associated task in **model-hub** that is guaranteed to work with
+Determined and verified for correctness. See the table below for a
+summary of the **model-hub** transformers examples:
+
+.. list-table::
+   :header-rows: 1
+
+   -  -  Task
```

Comment:
I would rather have Task, Dataset, Filename like in the existing https://docs.determined.ai/latest/examples.html. I don't think the Description field really adds much here, and like the idea of a tarball rather than specific links, but not sure if other folks' opinions here.